### DISCUSSION TITLE
Issues with uploading proof of enrollment for education benefits

### DISCUSSION THREAD

**Discussion by benvessely** [community/community#84812] [GitHub Education]
### Hi everyone,

I'm having trouble with figuring out how to upload my proof of academic status as required for the Github Student Development Pack. My student ID doesn't have a date, so I can't use that. I do have a schedule online that has my name and other relevant information, but there isn't a way to upload a document for this proof of academic status. Rather, we are only given the option to take a photo of the document with our computer camera, but I don't suppose that would do anything unless I can print the document and I won't have access to a printer for a bit. I also tried to complete this process on my phone so that I could just take a photo of the document with my phone camera, but it says "Use your computer's camera to take a picture now", and then it just shows a loading symbol and does nothing else when I click the "Take a picture" button below. Has anyone else had this issue? I will be able to print my document off eventually, but it seems like there should be a way to do this without having to print anything.

**Comment by Francis0763** [comment]
i have the same problem.
